core/src/core/block.rs

```diff
@@ -125,6 +127,13 @@ pub struct BlockHeader {
 	/// We can derive the kernel offset sum for *this* block from
 	/// the total kernel offset of the previous block header.
 	pub total_kernel_offset: BlindingFactor,
+	/// Total accumulated sum of kernel commitments since genesis block.
+	/// Should always equal the UTXO commitment sum minus supply.
+	pub total_kernel_sum: Commitment,
```

Comment:
Just clicked that we don't actually need (or want) the `total_output_sum` in here.
We can derive this from the total supply (block height).